gef-ui/server/routes/facilities/amendments/index.ts

```diff
@@ -20,4 +21,9 @@ router
   .all([validatePortalFacilityAmendmentsEnabled, validateToken, validateBank, validateRole({ role: [MAKER] })])
   .get(getFacilityValue);
 
+router
+  .route('/application-details/:dealId/facilities/:facilityId/amendments/do-you-have-a-facility-end-date')
+  .all([validatePortalFacilityAmendmentsEnabled, validateToken, validateBank, validateRole({ role: [MAKER] })])
+  .get(getDoYouHaveAFacilityEndDate);
+
```

Comment:
this route needs and the amendmentId I think. When I am on the New cover end Date which is the previous page the amendmentId has already created.

Reply:
Thanks @rkontogianni yes - at the time of PR creation no amendment ID was available, there is another sub task for this DTFS2-7714